cranelift-codegen/src/regalloc/branch_splitting.rs

```diff
@@ -34,6 +35,11 @@ struct Context<'a> {
     /// True if new blocks were inserted.
     has_new_blocks: bool,
 
+    /// Record whether newly inserted empty blocks should be inserted last, or
+    /// before the last, to avoid disturbing the expected control flow of
+    /// `fallthroug_return` statements.
+    has_fallthrough_return: Option<bool>,
```

Comment:
Maybe explain that `None` means not yet calculated, while `Some` means the inner bool determines if there is an `has_fallthrough_return`.

Reply:
Thank, I added 2 statements to this comment to clarify that the Option wraps the computed value.